### PR TITLE
Add documentation for the ZoneMinder runstate sensor and service

### PR DESCRIPTION
**Pull request by rohankapoorcom**
**Description:**
This is a re-implementation of @jantman's PR in #5655 to go with a re-implementation of ZoneMinder run states in home-assistant.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17198

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @rohankapoorcom,
The current version of the documentation looks fine.

File: source/_components/sensor.zoneminder.markdown [hunk 1]
```diff
@@ -14,7 +14,7 @@ ha_iot_class: "Local Polling"
 ---
 
 
-The `zoneminder` sensor platform lets you monitor the current state of your [ZoneMinder](https://www.zoneminder.com) install including the number of events and the current state of the cameras.
+The `zoneminder` sensor platform lets you monitor the current state of your [ZoneMinder](https://www.zoneminder.com) install including the number of event, the current state of the cameras and ZoneMinder's current run state.
```
Comment (MartinHjelmare) [inline source/_components/sensor.zoneminder.markdown]: ... including the number of events...

**Review by MartinHjelmare — COMMENTED** [review]
Please revert the access mode changes on the files.

See below for a small wording change.

**Comment by rohankapoorcom** [comment]
@MartinHjelmare Wording fixed - thanks for the typo catch.

I'm not seeing an access mode change on my machine. What's the change you are seeing?

**Comment by MartinHjelmare** [comment]
Go to changes page here in the PR:
```
 source/_components/sensor.zoneminder.markdown 100644 → 100755
```

**Comment by rohankapoorcom** [comment]
Thanks, I see it now. I think it's fixed. I'm assuming it's something weird with my file system setup (NTFS windows mounted into ubuntu via the windows subsystem for linux). I guess I have to be careful which git client I'm using.

**Review by MartinHjelmare — APPROVED** [review]
Nice!

**Pull request merged**